--- avogadro/qtplugins/plugindownloader/zipextracter.org.h	2020-02-19 11:35:22.297237093 +0100
+++ avogadro/qtplugins/plugindownloader/zipextracter.h		2020-02-19 11:35:38.253251321 +0100
@@ -14,8 +14,8 @@
 
 ******************************************************************************/
 
-#include "archive.h"
-#include "archive_entry.h"
+#include <archive.h>
+#include <archive_entry.h>
 #include <QtCore/QList>
 #include <string>
 
