--- avogadro/qtplugins/quantuminput/CMakeLists.orig.txt	2016-12-01 22:17:48.000000000 +0100
+++ avogadro/qtplugins/quantuminput/CMakeLists.txt	2017-03-12 15:17:46.126263369 +0100
@@ -41,7 +41,7 @@
   endif()
 
   install(PROGRAMS ${_generators}
-    DESTINATION "${INSTALL_LIBRARY_DIR}/avogadro2/scripts/inputGenerators/")
+    DESTINATION "${INSTALL_LIBRARY_DIR}/python${PYTHON_VERSION}/site-packages/avogadro2/scripts/inputGenerators/")
 else()
   message("Avogadro Generators not found at ${_prefix}!")
 endif()
--- avogadro/qtplugins/scriptfileformats/CMakeLists.orig.txt	2016-12-01 22:17:48.000000000 +0100
+++ avogadro/qtplugins/scriptfileformats/CMakeLists.txt	2017-03-12 15:30:07.829240927 +0100
@@ -24,4 +24,4 @@
 )
 
 install(PROGRAMS ${format_scripts}
-  DESTINATION "${INSTALL_LIBRARY_DIR}/avogadro2/scripts/formatScripts/")
+  DESTINATION "${INSTALL_LIBRARY_DIR}/python${PYTHON_VERSION}/site-packages/avogadro2/scripts/formatScripts/")
